### ISSUE TITLE
Dashboard enhancements 

### ISSUE THREAD

**Issue by mjherzog** [nexB/scancode-workbench#381]
This enhancement request is to update the Dashboards to consolidate information by primary topic. The changes are:

1. Create a new Copyright Dashboard:
-- Move "Unique Copyright Holder Detected" and Copyright Holder pie-chart here from the Fileinfo Dashboard
-- Move Copyright Information bar charts here from the current Chart Summary View
-- We might want 3 "columns" for Copyright Holders, Statements and Authors where each column has the Unique Detected number on top, with pie-chart below and button to open the corresponding bar chart below the pie-chart

2. Update FileInfo Dashboard:
-- Add Number of Binary Files box on the right (replacing Unique Copyright Holders Detected)
-- Add Mime Types pie-chart to the FileInfo Dashboard on the right (replacing Copyright Holders)

3. License Info Dashboard:
-- Add License Category pie-chart
-- Move License Expressions bar chart here from the current Chart Summary View
-- License Expressions pie-chart currently shows License Key values - not sure how to interpret this versus License Keys pie-chart - do we need both?

4. Remove Bar Chart Summary View after moving charts to Copyright or License Dashboards.

NB - I am not sure how to best present the Bar Charts in their new locations. The current first page display for Bar Charts is very "cryptic" so it would be nicer to display each option directly, but understand that the charts do n

**Comment by majurg** [comment]
I think consolidation into three dashboards and removing the chart summary views is a good way to go. 

We could have the bar charts under their respective pie chart graphs? that way the various bar charts are covered for each dash board. Alternatively we could have a single bar chart with selective options like we have now.

**Comment by mjherzog** [comment]
I think that separate bar charts is much easier to understand - perhaps there is a way to show the first ten bars or so in preview mode on the main page and click somewhere to open up the bar chart for all the data?

**Comment by nitin10s** [comment]
I agree that the separate bar chart option makes it much more accessible. In my opinion, we should not remove the chart summary. What we can do is add an " Open Bar Chart" option under the Pie chart that redirects the user to the Bar Chart Summary View. This would make it more convenient to the user. What'd you say? @majurg @mjherzog

**Comment by mjherzog** [comment]
Replaced with more focused issues #578 and #580
